code/game/machinery/washing_machine.dm

```diff
@@ -199,7 +199,7 @@
 	set src in usr.loc
 
 	sleep(20)
-	if(state in list(1,3,6) )
+	if(state in list(1,3,6))
```

Comment:
While we're here, could you space these up?

And the other few changes you made in this file too?

Reply:
Out of scope